packages/issuer-sdk-js/README.md

```diff
@@ -161,29 +161,25 @@ import { createCredentialByGrant, encryptionPublicKey } from "@idos-network/issu
 import issuerConfig from "./issuer-config.js";
 
 const credential = {
-  // These three fields are public and are designed to assist dApps that rely
-  // on your credentials with selecting the appropriate credential when
-  // acquiring an Access Grant.
-  //
-  // You should decide on what's the most helpful content they can have. These
-  // three are just an example.
-  credential_type: "human",
-  credential_level: "human",
-  credential_status: "approved",
-
-  // This is a string decided by you. Be sure to always use the same one to
-  // make yourself discoverable by dApps.
-  issuer: "MyCoolIssuer",
-
-  // user id of the human who is creating the credential.
-  human_id: session.user.humanId,
-
+  // The user id of the human who is creating the credential.
+  humanId: session.user.humanId,
+  // Fields in the `publicNotes` field are fields that are public and can be used by dApps to select the appropriate credential when acquiring an Access Grant.
+  // You should decide on what's the most helpful content they can have.
+  publicNotes: JSON.stringify({
+    type: "human",
+    level: "human",
+    status: "approved", 
+    
+    // make yourself discoverable by dApps.
+    issuer: "MyCoolIssuer",
```

Comment:
`issuer` is meant to be `HexCoded.encode(signingKeyPair.publicKey, true)` at the credential (not its public_notes).